docs/org/reference/language/control.org

```diff
@@ -0,0 +1,27 @@
+* Control structures
+
+** Case
+
+A case expression has the following syntax:
+
+#+begin_example
+case value
+| pat1 := branch1
+..
+| patN := branchN
+#+end_example
+
+For example, one can evaluate the following expression in the REPL:
+#+begin_example
+Stdlib.Prelude> case 1, 2 | suc _, zero := 0 | suc _, suc x := x | _ := 19
```

Comment:
I feel like this will be a bit hard to parse for a newcomer. I would prefer to see a simpler `case` expression in the repl or write it multiline outside of the repl (or both)

Reply:
Okay, I simplified the case expression by removing the pair